test/arcade/potts/agent/cell/PottsCellTest.java

```diff
@@ -79,6 +76,35 @@ public class PottsCellTest {
     static EnumSet<Region> regionList;
     
     static MiniBox parametersMock;
+
+    static class PottsCellMock extends PottsCell {
+        PottsCellMock(int id, int parent, int pop, CellState state, int age, int divisions,
+                      Location location, boolean hasRegions, MiniBox parameters,
+                      double criticalVolume, double criticalHeight,
+                      EnumMap<Region, Double> criticalRegionVolumes,
+                      EnumMap<Region, Double> criticalRegionHeights) {
+            super(id, parent, pop, state, age, divisions, location, hasRegions, parameters,
+            criticalVolume, criticalHeight, criticalRegionVolumes, criticalRegionHeights);
+
+        }
+
+        @Override
+        public PottsCellMock make(int newID, CellState newState, Location newLocation,
+                              MersenneTwisterFast random) {
+            return new PottsCellMock(newID, id, pop, newState, age, divisions, newLocation,
+                            hasRegions, parameters, criticalVolume, criticalHeight,
+                            criticalRegionVolumes, criticalRegionHeights);
+        }
+        
+        @Override
+        public void setStateModule(CellState newState) {
+            if (newState == State.PROLIFERATIVE) {
+                module = new PottsModuleProliferationSimple(this);
```

Comment:
@jessicasyu how do you feel about this as a solution that allows us to check that a module and state are set in the pottsCellContainer in the convert function?